sdk/compute/Microsoft.Azure.Management.Compute/src/Microsoft.Azure.Management.Compute.csproj

```diff
@@ -31,4 +31,4 @@
   <!-- Please do not move/edit code below this line -->
```

Comment:
Should we make some updates in this file on the version number and release notes?

Reply:
we should. you're right. let me do that

Reply:
all updated